src/Game/Game.cpp

```diff
@@ -448,5 +446,19 @@ Audio::Mixer* Game::mixer()
     return _mixer.get();
 }
 
+Event::Dispatcher& Game::eventDispatcher()
+{
+    return _eventDispatcher;
+}
+
+}
+
+namespace Event
+{
+// static
+Dispatcher* Dispatcher::getInstance()
+{
+    return &Game::getInstance()->eventDispatcher();
+}
```

Comment:
Why do we need EventDispatcher outside of Game lifetime? The game holds all objects and all event-related logic. I think it would be OK to put EventDispatcher inside a Game.